handler/src/main/java/io/netty/handler/timeout/IdleStateHandler.java

```diff
@@ -34,6 +35,9 @@
  * Triggers an {@link IdleStateEvent} when a {@link Channel} has not performed
  * read, write, or both operation for a while.
  *
+ * The {@link IdleStateHandler} should be placed as first {@link ChannelHandler}
```

Comment:
Putting it after the decoder will force the attacker to deliver a HTTP request within the specified threshold.